examples/src/main/java/com/example/ftclibexamples/PurePursuitSample.java

```diff
@@ -0,0 +1,67 @@
+package com.example.ftclibexamples;
+
+import com.arcrobotics.ftclib.command.CommandOpMode;
+import com.arcrobotics.ftclib.command.OdometrySubsystem;
+import com.arcrobotics.ftclib.command.PurePursuitCommand;
+import com.arcrobotics.ftclib.drivebase.MecanumDrive;
+import com.arcrobotics.ftclib.hardware.motors.Motor;
+import com.arcrobotics.ftclib.hardware.motors.MotorEx;
+import com.arcrobotics.ftclib.kinematics.HolonomicOdometry;
+import com.arcrobotics.ftclib.purepursuit.waypoints.EndWaypoint;
+import com.arcrobotics.ftclib.purepursuit.waypoints.GeneralWaypoint;
+import com.arcrobotics.ftclib.purepursuit.waypoints.StartWaypoint;
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.Disabled;
+
+@Autonomous
+@Disabled
+public class PurePursuitSample extends CommandOpMode {
+
+    // define our constants
+    static final double TRACKWIDTH = 13.7;
+    static final double TICKS_TO_INCHES = 15.3;
```

Comment:
This should be computed, not a static variable.

Reply:
![image](https://user-images.githubusercontent.com/17654209/95811634-b774eb80-0ce1-11eb-96ac-2b8865daad77.png)
Does something like this work?

Reply:
Yup, another thing which could be done is just having wheel rad and ticks and then figure it out based on that.